system_tests/program_test.go

```diff
@@ -1242,6 +1242,134 @@ func testSdkStorage(t *testing.T, jit bool) {
 	check()
 }
 
+func TestStylusPrecompileMethodsSimple(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	builder := NewNodeBuilder(ctx).DefaultConfig(t, true)
+	cleanup := builder.Build(t)
+	defer cleanup()
+
+	arbOwner, err := pgen.NewArbOwner(types.ArbOwnerAddress, builder.L2.Client)
+	Require(t, err)
+	arbDebug, err := pgen.NewArbDebug(types.ArbDebugAddress, builder.L2.Client)
+	Require(t, err)
+	arbWasm, err := pgen.NewArbWasm(types.ArbWasmAddress, builder.L2.Client)
+	Require(t, err)
+
+	ensure := func(tx *types.Transaction, err error) *types.Receipt {
+		t.Helper()
+		Require(t, err)
+		receipt, err := EnsureTxSucceeded(ctx, builder.L2.Client, tx)
+		Require(t, err)
+		return receipt
+	}
+
+	ownerAuth := builder.L2Info.GetDefaultTransactOpts("Owner", ctx)
+	ensure(arbDebug.BecomeChainOwner(&ownerAuth))
+
+	// ArbOwner precompile methods
+	testConst := uint16(10)
+	ensure(arbOwner.SetInkPrice(&ownerAuth, uint32(testConst)))
+	ensure(arbOwner.SetWasmMaxStackDepth(&ownerAuth, uint32(testConst)))
+	ensure(arbOwner.SetWasmFreePages(&ownerAuth, testConst))
+	ensure(arbOwner.SetWasmPageGas(&ownerAuth, testConst))
+	ensure(arbOwner.SetWasmPageLimit(&ownerAuth, testConst))
+	// Setting low values of gas and cached parameters ensures when MinInitGas is called on ArbWasm precompile,
+	// the returned values would be programs.MinInitGasUnits and programs.MinCachedGasUnits
+	ensure(arbOwner.SetWasmMinInitGas(&ownerAuth, 1, 1))
+	ensure(arbOwner.SetWasmInitCostScalar(&ownerAuth, uint64(testConst)))
+	expectedExpiryDays := uint16(1)
+	ensure(arbOwner.SetWasmExpiryDays(&ownerAuth, expectedExpiryDays))
+	ensure(arbOwner.SetWasmKeepaliveDays(&ownerAuth, 0))
+	ensure(arbOwner.SetWasmBlockCacheSize(&ownerAuth, testConst))
+
+	// ArbWasm precompile methods
+	wasm, _ := readWasmFile(t, rustFile("keccak"))
+	codehash := crypto.Keccak256Hash(wasm)
+	programAddress := deployContract(t, ctx, ownerAuth, builder.L2.Client, wasm)
+
+	activateAuth := ownerAuth
+	activateAuth.Value = oneEth
+	ensure(arbWasm.ActivateProgram(&activateAuth, programAddress))
+
+	bcs, err := arbWasm.BlockCacheSize(nil)
```

Comment:
nitpick: how about testing getters right after using the related setter?
I mean, testing `arbWasm.BlockCacheSize(nil)` right after calling `ensure(arbOwner.SetWasmBlockCacheSize(&ownerAuth, testConst))`.
This can improve readability.